source/game/system/CourseMap.hh

```diff
@@ -23,12 +25,14 @@ class CourseMap : EGG::Disposer {
 public:
     void init();
     [[nodiscard]] MapdataCannonPointAccessor *parseCannonPoint(u32 sectionName);
+    [[nodiscard]] MapdataCheckPointAccessor *parseCheckPoint(u32 sectionName);
     [[nodiscard]] MapdataGeoObjAccessor *parseGeoObj(u32 sectionName);
     [[nodiscard]] MapdataStageInfoAccessor *parseStageInfo(u32 sectionName);
     [[nodiscard]] MapdataStartPointAccessor *parseStartPoint(u32 sectionName);
```

Comment:
`parseCheckPoint` can actually be `const`. At that rate, ALL of these can actually be `const`. Would you mind adding them all here? Just for sake of consistency (I don't want to have you adjust the function you added without fixing the others)
```suggestion
    [[nodiscard]] MapdataCannonPointAccessor *parseCannonPoint(u32 sectionName) const;
    [[nodiscard]] MapdataCheckPointAccessor *parseCheckPoint(u32 sectionName) const;
    [[nodiscard]] MapdataGeoObjAccessor *parseGeoObj(u32 sectionName) const;
    [[nodiscard]] MapdataStageInfoAccessor *parseStageInfo(u32 sectionName) const;
    [[nodiscard]] MapdataStartPointAccessor *parseStartPoint(u32 sectionName) const;
```